<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="COMV-9549" failures="1" tests="3" name="Test" time="13.838" errors="0" timestamp="2025-01-10T19:17:39 IST">
  <testcase classname="day46.OrangeHRM" name="testLogo" time="0.055"/>
  <testcase classname="day46.OrangeHRM" name="testAppURL" time="0.019">
    <failure type="java.lang.AssertionError" message="expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth/logins] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]">
      <![CDATA[java.lang.AssertionError: expected [https://opensource-demo.orangehrmlive.com/web/index.php/auth/logins] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at day46.OrangeHRM.testAppURL(OrangeHRM.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testAppURL -->
  <testcase classname="day46.OrangeHRM" name="verifyHomePageTitle" time="0.0">
    <skipped/>
  </testcase> <!-- verifyHomePageTitle -->
</testsuite> <!-- Test -->
